commons_boundaries/lib.py

```diff
@@ -10,3 +13,12 @@ def write_csv_from_shp(shp_path, csv_path):
             writer.writeheader()
             for feature in shp:
                 writer.writerow(feature['properties'])
+
+
+def proj4_from_shp(shp_path):
+    shp_path = Path(shp_path)
+    crs = to_string(fiona.open(str(shp_path)).crs)
```

Comment:
I think you can just simplify this to `return to_string(fiona.open(str(shp_path)).crs)` - if the `.prj` file is missing it looks like `fiona.open(str(shp_path)).crs` returns `{}` and `to_string` turns that into `''`